src/components/Item.tsx

```diff
@@ -1,10 +1,18 @@
-const Item = (): JSX.Element => {
+interface ItemProps {
+  product: string
+  price: number,
+  link: string,
+  img: string
+}
+
+const Item = (props: {data: ItemProps}): JSX.Element => {
```

Comment:
```suggestion
const Item = (data: ItemProps): JSX.Element => {
```
このようにpropsを展開すれば`props.`という書き方をしなくて良いのでコードが見やすくなります:bow:
実際に業務では`props.product `みたいな書き方はしていないです！

Reply:
教材の内容わからないので的外れな発言かもしれませんが、僕なら`type dataProps = { data: ItemProps }`という型を定義して`const Item = ({data}: dataProps) => {}`ような書き方をするかなと思いました:bow: